install-poc.sh

```diff
@@ -5,10 +5,20 @@ defaultSidecarVersion="v2.34.0"
 ###
 if [ -z "$containerRegistry" ]; then
     containerRegistry="gcr.io/cyralinc"
+else
+    echo "containerRegistry enviroment variable found, using that"
```

Comment:
What if we make it clearer by stating what is being here? Same would apply to the other messages.
```suggestion
    echo "Using container registry: $containerRegistry"
```

Reply:
slight tweak to retain the variable name in case they are unsure how things are automatically showing up, applied to others vars as well